source/XCodePlugin/Plugin.cpp

```diff
@@ -246,6 +246,18 @@ void _FirebaseUnAuth(void* firebase) {
     jniFirebase->UnAuth();
 }
 
+void _FirebaseGoOffline(void* firebase) {
+    if (!firebase) return;
```

Comment:
you probably dont need the firebase instance if these are statics.
But if it doesn't hurt... its not bad to keep the pattern the same.

Were you able to recompile the libs to test?  It's a bit complicated as the bundle for editor/mac needs to be compiled in XCode and the same code is compiled in Visual Studio for windows/editor.